pandas/core/base.py

```diff
@@ -507,7 +507,7 @@ def is_any_frame():
                            for r in compat.itervalues(result))
 
             if isinstance(result, list):
-                return concat(result, keys=keys, axis=1), True
```

Comment:
thre are a bunch more concats in this same section. basically they control the resulting ordeing of the aggregation. I guess ``sort=True`` is fine here (for the other cases).

Reply:
or maybe these should be ``sort=False``, not sure what is actually affected (as these might be aligned ops already)